<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I hate TC2 so much</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QCheckBox" name="cb_showCurrentGear">
      <property name="text">
       <string>Display current gear engaged</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_profile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>User profiles</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="hlay_profileComboBox">
         <item>
          <widget class="QLabel" name="lbl_disp_selectProfile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select profile :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_selectDevice"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="hlay_pad">
      <item>
       <widget class="QLabel" name="lbl_pad_LB">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(180,0,0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_pad_RB">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(180,0,0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="gb_settings">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tb_settings">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QScrollArea" name="tb_settingsPage1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>355</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollArea#tb_settingsPage1{
	
	background-color: rgba(255, 255, 255, 0);
}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>In game</string>
           </attribute>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>339</width>
              <height>495</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#scrollAreaWidgetContents{
	
	background-color: rgba(255, 255, 255, 0);
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QPushButton" name="pb_selectKey_GReverse">
               <property name="text">
                <string>Reverse</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1" rowspan="4">
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLabel" name="lbl_G5">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="14" column="0">
              <widget class="QPushButton" name="pb_selectKey_SeqDown">
               <property name="text">
                <string>Gear down (Seq)</string>
               </property>
              </widget>
             </item>
             <item row="13" column="2">
              <widget class="QLabel" name="lbl_seqUp">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QPushButton" name="pb_selectKey_G7">
               <property name="text">
                <string>Gear 7</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_disp_selectKey">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Select key</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="pb_selectKey_G4">
               <property name="text">
                <string>Gear 4</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="lbl_disp_currKey">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Current key</string>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QPushButton" name="pb_selectKey_SeqUp">
               <property name="text">
                <string>Gear up (Seq)</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QLabel" name="lbl_G6">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="pb_selectKey_G3">
               <property name="text">
                <string>Gear 3</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="lbl_GClutch">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="17" column="0">
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="18" column="2">
              <widget class="QSpinBox" name="sb_gearDelay">
               <property name="toolTip">
                <string>How long the key is going to be pressed down</string>
               </property>
               <property name="maximum">
                <number>1200</number>
               </property>
              </widget>
             </item>
             <item row="11" column="0" colspan="3">
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="14" column="2">
              <widget class="QLabel" name="lbl_seqDown">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pb_selectKey_G1">
               <property name="text">
                <string>Gear 1</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="lbl_G4">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="19" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="lbl_G2">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QLabel" name="lbl_G7">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="16" column="0" colspan="3">
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="15" column="0">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pb_selectKey_GClutch">
               <property name="text">
                <string>Neutral - Clutch</string>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="lbl_GReverse">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QPushButton" name="pb_selectKey_G5">
               <property name="text">
                <string>Gear 5</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="11">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="lbl_G3">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="pb_selectKey_G6">
               <property name="text">
                <string>Gear 6</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="lbl_G1">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pb_selectKey_G2">
               <property name="text">
                <string>Gear 2</string>
               </property>
              </widget>
             </item>
             <item row="18" column="0">
              <widget class="QLabel" name="lbl_disp_gearDelay">
               <property name="toolTip">
                <string>How long the key is going to be pressed down</string>
               </property>
               <property name="text">
                <string>Gear key press delay (ms)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tb_settingsPage2">
           <attribute name="title">
            <string>Controller</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="9" column="0">
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <spacer name="verticalSpacer_9">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="9" column="1" rowspan="2">
             <widget class="Line" name="line_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pb_selectButton_GDown">
              <property name="text">
               <string>Gear Down</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="3">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>297</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="lbl_disp_currKey_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Current key</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QPushButton" name="pb_selectButton_switchMode">
              <property name="text">
               <string>Switch mode
Sequential / Clutch</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QLabel" name="lbl_btn_switchMode">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pb_selectButton_GUp">
              <property name="text">
               <string>Gear Up</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="lbl_btn_GDown">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_disp_selectButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Select key</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="Line" name="line_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pb_selectButton_firstGear">
              <property name="text">
               <string>1st gear</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="8" column="0" colspan="3">
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="lbl_btn_GUp">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="lbl_btn_firstGear">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_10">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" rowspan="4">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="1" rowspan="3">
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_selectKey_resetDefault">
          <property name="text">
           <string>Default settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QDockWidget" name="dockConsole">
       <widget class="QWidget" name="dockWidgetContents">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="debug::LogConsole" name="consoleLog"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QToolButton" name="toolB_help">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="assets/resources.qrc">
        <normaloff>:/img/img/helpIcon.png</normaloff>:/img/img/helpIcon.png</iconset>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonIconOnly</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuAdvanced">
     <property name="title">
      <string>Advanced</string>
     </property>
     <addaction name="action_ShowDevconsole"/>
    </widget>
    <addaction name="menuAdvanced"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
    <addaction name="separator"/>
    <addaction name="action_checkUpdates"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_ShowDevconsole">
   <property name="text">
    <string>Show Dev console</string>
   </property>
   <property name="toolTip">
    <string>Display developper console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_checkUpdates">
   <property name="text">
    <string>Check updates</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>debug::LogConsole</class>
   <extends>QTextBrowser</extends>
   <header>Debug/logConsole.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
